src/elements/content-sidebar/activity-feed/comment/messages.js

```diff
@@ -62,6 +57,11 @@ const messages = defineMessages({
         defaultMessage: 'See {repliesToLoadCount, plural, one {# reply} other {# replies}}',
         description: 'Text to show to get more replies of comment or annotation',
     },
+    inlineCommentAnnotationAccessibleSVGTitle: {
+        id: 'be.contentSidebar.activityFeed.comment.IconAnnotation',
```

Comment:
minor - the ID was changed:
```suggestion
    inlineCommentAnnotationAccessibleSVGTitle: {
        id: 'be.contentSidebar.activityFeed.comment.inlineCommentAnnotationAccessibleSVGTitle',
```